lib/chef/provider/windows_task.rb

```diff
@@ -20,26 +20,153 @@
 require "rexml/document"
 require "iso8601"
 require "chef/mixin/powershell_out"
+require "chef/provider"
+require "win32/taskscheduler"
```

Comment:
We need to get this dep into the gemspec

Reply:
yes I will be committing those file